Tests/Orso.Arpa.Api.Tests/IntegrationTests/ProjectsControllerTests.cs

```diff
@@ -27,5 +31,112 @@ public async Task Should_Get_All()
+                CreatedBy = _staff.DisplayName,
+            };
+
+            // Act
+            HttpResponseMessage responseMessage = await _authenticatedServer
+                .CreateClient()
+                .AuthenticateWith(_staff)
+                .PostAsync(ApiEndpoints.ProjectsController.Post(), BuildStringContent(createDto));
+
+            // Assert
+            responseMessage.StatusCode.Should().Be(HttpStatusCode.Created);
+            ProjectDto result = await DeserializeResponseMessageAsync<ProjectDto>(responseMessage);
+
+            result.Should().BeEquivalentTo(expectedDto, opt => opt.Excluding(r => r.CreatedAt).Excluding(r => r.Id));
+            result.Id.Should().NotBeEmpty();
+            result.CreatedAt.Should().BeBefore(DateTime.UtcNow);
+            result.CreatedAt.Should().BeAfter(DateTime.MinValue);
+        }
+
+        [Test, Order(107)]
+        public async Task Should_Modify()
+        {
+            // Arrange
+            Project projectToModify = ProjectSeedData.HoorayForHollywood;
+
+            var modifyDto = new ProjectModifyDto
+            {
+                Title = "changed" + projectToModify.Title,
+                Description = "changed" + projectToModify.Description,
+            };
+
+            // Act
+            HttpResponseMessage responseMessage = await _authenticatedServer
+                .CreateClient()
+                .AuthenticateWith(_staff)
+                .PutAsync(ApiEndpoints.ProjectsController.Put(projectToModify.Id), BuildStringContent(modifyDto));
+
+            // Assert
+            responseMessage.StatusCode.Should().Be(HttpStatusCode.NoContent);
+
+            // check now, if modification really did make it into the database. Fetch the (now modified) project again via GetById
+
+            // Act
+            responseMessage= await _authenticatedServer
+                .CreateClient()
+                .AuthenticateWith(_performer)
+                .GetAsync(ApiEndpoints.ProjectsController.Get(projectToModify.Id));
+
+            // Assert
+            responseMessage.StatusCode.Should().Be(HttpStatusCode.OK);
+            ProjectDto result = await DeserializeResponseMessageAsync<ProjectDto>(responseMessage);
+            result.Should().BeEquivalentTo(modifyDto, opt => opt.Excluding(r => r.Id));
+            result.ModifiedBy.Should().Be(_staff.DisplayName);
+            result.ModifiedAt.Should().BeBefore(DateTime.UtcNow);
+            result.ModifiedAt.Should().BeAfter(DateTime.MinValue);
+        }
+
+        [Test, Order(10004)]
+        public async Task Should_Delete()
+        {
+            // Arrange
+            Project projectToDelete = ProjectSeedData.HoorayForHollywood;
+
+            // Act
+            HttpResponseMessage responseMessage = await _authenticatedServer
+                .CreateClient()
+                .AuthenticateWith(_staff)
+                .DeleteAsync(ApiEndpoints.ProjectsController.Delete(projectToDelete.Id));
```

Comment:
Über das Löschen müssen wir denk ich nochmal detailliert sprechen (betrifft auch die Appointments, an denen ich gerade dran bin), um zu klären, welche Daten kaskadierend mitgelöscht werden sollten.

Reply:
Können wir das in eine separate UserStory auslagern?

Reply:
Ja klar, wenn ich damit durch bin, kann es gut sein, dass es für die Projects schon automatisch funktioniert